crates/criticalup-cli/src/commands/install.rs

```diff
@@ -175,39 +218,95 @@ fn check_for_package_dependencies(verified_release_manifest: &Release) -> Result
     Ok(())
 }
 
-#[test]
-fn dependencies_check() {
-    use criticaltrust::manifests::ReleasePackage;
-
-    let dependencies = vec!["dependency_a".to_string()];
-
-    let good = Release {
-        product: "ferrocene".to_string(),
-        release: "nightly-2024-02-28".to_string(),
-        commit: "123".to_string(),
-        packages: vec![ReleasePackage {
-            package: "awesome".to_string(),
-            artifacts: vec![],
-            dependencies: vec![],
-        }],
-    };
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use criticaltrust::manifests::{ReleaseArtifact, ReleasePackage};
+    use time::macros::datetime;
+
+    const PACKAGE_SHA256: &[u8] = &[
```

Comment:
Not a change request but you can avoid these blobs with hex_literal::hex I think in the future https://docs.rs/sha2/latest/sha2/index.html#usage

Reply:
Thank you!